controllers/goharbor/jobservice/jobservice.go

```diff
@@ -39,6 +40,7 @@ type Reconciler struct {
 
 // +kubebuilder:rbac:groups=goharbor.io,resources=jobservices,verbs=get;list;watch
 // +kubebuilder:rbac:groups=goharbor.io,resources=jobservices/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=networking.k8s.io,resources=networkpolicies,verbs=get;list;watch;create;update;patch;delete
```

Comment:
@sguyennet 

Why only some of the component reconcilers have RBAC settings and others are not?

Reply:
The ones without RBAC settings are the ones without network policies